packages/paste-token-designer/components/token-option/FontTokenInput.js

```diff
@@ -0,0 +1,26 @@
+import {Box} from '@twilio-paste/core/box';
+import {useUIDSeed} from '@twilio-paste/uid-library';
+import {Input} from '@twilio-paste/core/input';
+import {Label} from '@twilio-paste/core/label';
+
+// TODO
+export const FontTokenInput = ({bucket, tokenName, tokenValue, onChange}) => {
+  const seed = useUIDSeed();
```

Comment:
nit: change to `inputId` or something like that.

Reply:
It's a uid seed though :)